src/templates/static/legal/metatrader/terms_and_conditions.jsx

```diff
@@ -1,446 +1,660 @@
+                    <li>{it.L('The Client acknowledges that it is their responsibility to monitor their Account Balance and Margin Requirement, and the Company is not under any obligation to keep the Client informed (i.e. to make a Margin Call).')}</li>
                 </ol>
             </li>
-            <li>{it.L('Margin Requirement and Calculation')}
-                <ol>
-                    <li>{it.L('The Margin Requirement is a percentage of the value of the Open Positions and will continue to fluctuate in accordance with the value of the Open Positions throughout the duration of the relevant contracts (the "Contract Value").')}</li>
-                    <li>{it.L('Margin Requirements can increase/decrease based on the aggregated size of your Open Position determined by the volume (number of lots), contract size (value per point in the original currency of the contract), market price, and margin rate.')}</li>
-                    <li>{it.L('We may modify Margin Requirements for any Open Positions or new Transactions entered into, at our sole discretion. Formulas for Margin Requirements published on the website are indicative only and may be changed by the Company at its discretion at any time, due to changing market conditions or other factors.')}</li>
-                    <li>{it.L('Whilst we endeavor to close out your Open Positions if and when the Margin Level for your MT5 Real Account reaches or falls below the Stop Out Level, we do not guarantee that your Open Positions will be closed when the Margin Level for your MT5 Real Account reaches the Stop Out Level.')}</li>
-                    <li>{it.L('For fully covered accounts, no margin will be charged on open positions. However, swaps and other applicable fees can cause your Account Equity to turn negative. In that case the Stop Out process will be triggered.')}</li>
-                </ol>
+            <li>{it.L('Margin Requirement and calculation')}
+                <p>
+                    <ol>
+                        <li>{it.L('Margin Requirement is the amount that the Client needs to have in their account prior to entering into a trade and it is a percentage of the value of that trade. Margin Requirement will continue to increase/decrease in accordance with the volume and direction of the Open Positions.')}</li>
+                        <li>{it.L('The Company may modify Margin Requirements for any Open Positions or new Orders at its sole discretion. Formulas for Margin Requirements published on the website are indicative only and may be changed by the Company at its discretion at any time, due to changing market conditions or other factors.')}</li>
+                        <li>{it.L('Whilst the Company endeavours to close out the Client\'s Open Positions if and when the Margin Level for their MT5 Real Account reaches or falls below the Stop Out Level.')}</li>
```

Comment:
```suggestion
                        <li>{it.L('The Company endeavours to close out the Client\'s Open Positions if and when the Margin Level for their MT5 Real Account reaches or falls below the Stop Out Level.')}</li>
```

Reply:
@Khalidxx Terribly sorry, Khalid: Line 198 is the rest of line 197. The two should be merged as: 'Whilst the Company endeavours to close out the Client\'s Open Positions if and when the Margin Level for their MT5 Real Account reaches or falls below the Stop Out Level, the company does not guarantee that the Client\'s Open Positions will be closed when the Margin Level for their MT5 Real Account reaches the Stop Out Level.' Line 198 should then be deleted.

Reply:
Alright